api/envoy/api/v2/rds.proto

```diff
@@ -44,7 +45,25 @@ service RouteDiscoveryService {
   }
 }
 
-// [#comment:next free field: 9]
+// * HTTP :ref:`router filter <config_http_filters_router>`is
+// Virtual Host Discovery Service (VHDS) is used to dynamically update the list of virtual hosts for
+// a given RouteConfiguration. If VHDS is configured a virtual host list update will be triggerred
+// during an http request processing if a route for the request cannot be resolved. The
```

Comment:
"during the processing of an HTTP request if..."

Reply:
fixed